### PR TITLE
Style Tenant Security Tab

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2021-06-28 at 11 50 25 AM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx [hunk 1]
```diff
@@ -110,13 +105,13 @@ const TenantSecurity = ({
...
-        id: Date.now().toString(),
-        key: "",
-      },
+      // {
+      //   cert: "",
+      //   encoded_cert: "",
+      //   encoded_key: "",
+      //   id: Date.now().toString(),
+      //   key: "",
+      // },
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx:114]: Please remove this

File: portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx [hunk 2]
```diff
@@ -125,36 +120,30 @@ const TenantSecurity = ({
...
-      id: Date.now().toString(),
-      key: "",
-    },
+    // {
+    //   cert: "",
+    //   encoded_cert: "",
+    //   encoded_key: "",
+    //   id: Date.now().toString(),
+    //   key: "",
+    // },
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx:129]: Please remove this

File: portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx [hunk 3]
```diff
@@ -125,36 +120,30 @@ const TenantSecurity = ({
...
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx:138]: Please remove this

**Review by bexsoft — CHANGES_REQUESTED** [review]
:/

File: portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx]: done

File: portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx]: deon

File: portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx (hunk 3 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/TenantDetails/TenantSecurity.tsx]: done

**Review by bexsoft — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Review by jinapurapu — APPROVED** [review]

**Review by adfost — APPROVED** [review]

**Pull request merged**